src/CoreBundle/Resources/views/snippets-cms/TCMSFieldWYSIWYG/cKEditor/editor.html.twig

```diff
@@ -59,6 +58,11 @@
             CKEDITOR.on('instanceReady', function () {
                 if (CKEDITOR.lang.de) CKEDITOR.lang.de.common.browseServer = 'CMS Quelle wählen';
                 else if (CKEDITOR.lang.en) CKEDITOR.lang.en.common.browseServer = 'Select CMS Source';
+
+                document.addEventListener('tableEditorBeforeSaveEvent', function(e) {
+                    let data = CKEDITOR.instances['{{ sEditorName }}'].getData();
```

Comment:
This is probably a const?

Reply:
removed this and inlined the code instead